device-server/src/main/kotlin/com/badoo/automation/deviceserver/host/HostFactory.kt

```diff
@@ -43,7 +43,8 @@ class HostFactory(
                 ),
                 simulatorLimit = config.simulatorLimit,
                 concurrentBoots = config.concurrentBoots,
-                wdaRunnerXctest = getWdaRunnerXctest(remote.isLocalhost(), wdaSimulatorBundle, REMOTE_WDA_BUNDLE_ROOT)
+                wdaRunnerXctest = getWdaRunnerXctest(remote.isLocalhost(), wdaSimulatorBundle, REMOTE_WDA_BUNDLE_ROOT),
+                initialEnvironmentVariables = config.environmentVariables
```

Comment:
It seems that we don't need `initialEnvironmentVariables` property, and it should not be part of the host node configuration.
In most cases, one can expect environment variables to be specific to the app being tested, so it is a client responsibility to set env variables on a simulator before or during tests.

Please remove `initialEnvironmentVariables` completly.

Reply:
Done.
#34